docs/source/basics/multimodal_datasets.rst

```diff
@@ -45,7 +45,7 @@ in the text, ``"<image>"`` for where to place the image tokens. This will get re
     from torchtune.models.llama3_2_vision import llama3_2_vision_transform
     from torchtune.datasets.multimodal import multimodal_chat_dataset
 
-    transform = Llama3VisionTransform(
+    model_transform = Llama3VisionTransform(
```

Comment:
there's a line below that still uses transform:

`print(transform.decode(tokenized_dict["tokens"], skip_special_tokens=False))`

Reply:
Fixed this line, thank you!